src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,84 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+When you're ready to go live with your template, you might want to use a custom domain name in place of the name provided by Koji.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+
+Koji has a few subdomains that you can use at no cost and with no additional configuration.
+These include:
```

Comment:
Grammar note... "these" and "this" should specify what they are referring to. In this case, I think you can get rid of "these" entirely, and simply make the lead-in part of the previous sentence. E.g., "... no additional configuration, including:"